nntrainer/src/conv2d_layer.cpp

```diff
@@ -370,6 +493,108 @@ int Conv2DLayer::conv2d(float *in, TensorDim indim, const float *kernel,
     }
     I++;
   }
+  return status;
+}
+
+int Conv2DLayer::conv2d_gemm(const float *mKernel, TensorDim kdim,
+                             Tensor const &in, TensorDim outdim,
+                             unsigned int const *mstride,
+                             unsigned int const *pad, float *out,
+                             unsigned int o_size, bool channel_mode) {
+  int status = ML_ERROR_NONE;
+  std::vector<float> inCol;
```

Comment:
IMO `inCol` does not need to be vector and name should be `in_col`

Reply:
fixed. thanks.